cmd/uninstall.go

```diff
@@ -16,15 +16,24 @@ var uninstallCmd = &cobra.Command{
 	Short: "Uninstall KubeArmor from a Kubernetes Cluster",
 	Long:  `Uninstall KubeArmor from a Kubernetes Clusters`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		err := install.K8sUninstaller(client, uninstallOptions)
-		return err
+		if uninstallOptions.Legacy {
+			if err := install.K8sLegacyUninstaller(client, uninstallOptions); err != nil {
+				return err
+			}
+		} else {
+			if err := install.K8sUninstaller(client, uninstallOptions); err != nil {
+				return err
+			}
+		}
```

Comment:
We need to auto fallback to legacy uninstallation, if we don't find helm release.

Most of the uninstallations today are going to be through legacy mode, it's going to be a bad UX/DX if we ask them to use legacy flag here

Reply:
Done :+1: